src/main/java/net/mcreator/ui/workspace/ModElementCodeDropdown.java

```diff
@@ -65,16 +66,24 @@ class ModElementCodeDropdown extends JPopupMenu {
 					listMenu.setBackground(((Color) UIManager.get("MCreatorLAF.LIGHT_ACCENT")).darker());
 					listMenu.setForeground((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"));
 					listMenu.setIconTextGap(8);
-					listMenu.setBorder(BorderFactory.createEmptyBorder(10, 0, 10, 0));
+					listMenu.setBorder(BorderFactory.createEmptyBorder(10, 0, 11, 0));
+
+					AtomicReference<Separator> lastSeparator = new AtomicReference<>(null);

```

Comment:
Each separator is added _**before**_ files generated for _**next**_ list item, so removing the last file separator will merge files generated for two last items in the list data.

Reply:
But now I get two separators below and it is really ugly:

![image](https://user-images.githubusercontent.com/16374228/209694914-31bb198e-3204-4c89-a4a4-098037978b00.png)

Please fix this.

Reply:
Apparently the last item in your list does not pass its generation conditions?

Reply:
I can fix it the same way you wanted the dropdown to show its _entry count_.

Reply:
Yep, condition fails, but if condition fails, there can't just be numerous of lines then:

![image](https://user-images.githubusercontent.com/16374228/209695359-b87ed30c-2b3b-4d6d-a380-7885fccd6ee1.png)

Well a fix needs to be found. Maybe what you suggested will work.

Reply:
✔️ Fixed